### PR TITLE
updated search route documentation.

### PR DESCRIPTION
**Pull request by Sripathm2**

### CODE REVIEW

File: CAM2CameraDatabaseAPIClient/client.py [hunk 1]
```diff
@@ -627,6 +627,15 @@ def search_camera(self, **kwargs):
...
         be achieved by calling this function multiple times with offest parameter.
 
+        Examples
+        --------
+            - Searching a camera of type 'IP' into the database with city as 'West Lafayette'
+            Create a keyword arguments dictionary which contains all the parameters
+            needed to search the camera:
+                kwargs = {'camera_type': 'ip', 'city': 'West Lafayette'}
+            Pass the keyword arguments dictionary to the search_camera function:
+                search_camera(**kwargs)
```
Comment (mush-zhang) [inline CAM2CameraDatabaseAPIClient/client.py]: you should first create a client say `a_client = Client('id', 'pass')` and `a_client.search_camera(**kwargs)`. Also, `a_client.search_camera(camera_type='ip', city='West Lafayette')` would also work.

File: CAM2CameraDatabaseAPIClient/client.py [hunk 2]
```diff
@@ -627,6 +627,15 @@ def search_camera(self, **kwargs):
         Each time, this function can return a maximum of 100 cameras. Getting more cameras can
         be achieved by calling this function multiple times with offest parameter.
 
+        Examples
+        --------
+            - Searching a camera of type 'IP' into the database with city as 'West Lafayette'
+            Create a keyword arguments dictionary which contains all the parameters
+            needed to search the camera:
+                kwargs = {'camera_type': 'ip', 'city': 'West Lafayette'}
```
Comment (mush-zhang) [inline CAM2CameraDatabaseAPIClient/client.py:635]: will this line show in the documentation "code-like"?

**Review by mush-zhang — CHANGES_REQUESTED** [review]

File: CAM2CameraDatabaseAPIClient/client.py (hunk 2 above)
Comment (Sripathm2) [reply CAM2CameraDatabaseAPIClient/client.py:635]: yes i think so.

File: CAM2CameraDatabaseAPIClient/client.py (hunk 1 above)
Comment (Sripathm2) [reply CAM2CameraDatabaseAPIClient/client.py]: done

**Review by jaydenC88 — APPROVED** [review]
Good job

File: CAM2CameraDatabaseAPIClient/client.py (hunk 2 above)
Comment (mush-zhang) [reply CAM2CameraDatabaseAPIClient/client.py:635]: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Comment (Sripathm2) [reply CAM2CameraDatabaseAPIClient/client.py:635]: @mush-zhang  yes it will i just double checked it with the link and the document that we have.

**Review by mush-zhang — APPROVED** [review]
Great!

**Pull request merged**